README.md

```diff
@@ -26,7 +26,15 @@ of tokens deposited.
 
 ## Install
 
-### Foundry
+### Node.js (recommended)
+
+Sablier V2 Core is available as a Node.js package:
+
+```shell
+yarn add @sablier/v2-core
+```
+
+### Foundry (slow)
```

Comment:
Let's remove "slow" - can be misunderstood to mean that Foundry is slow. The "recommended" note above is sufficient.